Directory.Build.props

```
@@ -11,6 +11,9 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <LangVersion>latest</LangVersion>
     <WarningLevel>9999</WarningLevel>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
```

Release or CI?

---

I guess it doesn't matter much. I would prefer Release since you can easily use it locally to double-check that you didn't miss any warnings.

---

sorry for being lazy, I meant 

```suggestion
  <PropertyGroup Condition="'$(Configuration)' == 'Release' Or '$(CI)' != ''">
```